### PR TITLE
Add Persian translation for index

### PR DESCRIPTION
**Pull request by peylight**

### CODE REVIEW

File: appendix.fa.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+RMS has a history of mistreating women and making them feel uncomfortable, unsafe, and unwelcome. For incidents relating to RMS and MIT, please see [0].
```
Comment (ehashman) [inline appendix.fa.md:1]: This file can be removed since it has not been translated, and we're linking to the English.

File: index.fa.md [hunk 2]
```diff
@@ -0,0 +1,2474 @@
...
+title: نامه سرگشاده ای برای برکناری ریچارد ام. استالمن از همه مناصب رهبری
+description: ما ، امضا کنندگان ، خواستار استعفا کل هیئت FSF و برکناری ریچارد ام. استالمن از تمام مناصب رهبری هستیم. ما دیگر به رنج رفتارهای وی ، دادن نقش رهبری به او یا ادامه پذیرفتن او و ایدئولوژی آسیب رسان و خطرناک او ادامه نخواهیم داد. 
+image: /assets/og-image.png
+twitter:
+  card: summary_large_image
+share_text: من از این فراخوان برای کنار رفتن کل هیئت FSF و برکناری ریچارد ام. استالمن از سمت های رهبری حمایت می کنم. 
+---
+
+{% include share-links.html %}
+
```
Comment (ehashman) [inline index.fa.md:11]: You can leave all but the title off the translated version.

File: index.fa.md [hunk 3]
```diff
@@ -0,0 +1,2474 @@
...
+تا کنون تحمل کافی عقاید و رفتارهای نفرت انگیز RMS وجود داشته است. ما نمی توانیم ادامه دهیم که یک نفر معنای کار ما را خراب کند. جوامع ما فضایی برای افرادی مانند ریچارد ام. استالمن ندارند و ما با رفتارهای وی ، دادن نقش رهبری به او و یا در غیر اینصورت پذیرفتن او و ایدئولوژی آسیب رسان و خطرناک او ، ادامه نخواهیم داد. 
+
+**ما خواهان حذف کل اعضای بنیاد نرم افزار آزاد هستیم.** اینها افرادی هستند که سالها RMS را فعال و توانمند کرده اند. آنها با اجازه دادن به او برای عضویت مجدد در شورای FSF این امر را دوباره نشان می دهند. وقت آن است که RMS از نرم افزار آزاد ، اخلاق فناوری ، حقوق دیجیتال و جوامع فناوری عقب نشینی کند ، زیرا او نمی تواند رهبری مورد نیاز ما را تأمین کند. **ما همچنین خواهان برکناری ریچارد ام. استالمن از کلیه مناصب رهبری ، از جمله پروژه GNU هستیم.**
```
Comment (ehashman) [inline index.fa.md:20]: For the first line: there does not appear to be a reference to the board of the Free Software Foundation. (I'm relying on machine translation). We want to remove all members of the FSF Board.

File: index.fa.md [hunk 4]
```diff
@@ -0,0 +1,2474 @@
...
+
+ما از افرادی که در موقعیتی هستند اصرار داریم که این کار را انجام دهند تا حمایت از بنیاد نرم افزار آزاد را متوقف کنند. از مشارکت در پروژه های مربوط به FSF و RMS خودداری کنید. در رویدادهای FSF ، یا رویدادهایی که از RMS و مارک عدم تحمل وی استقبال می کنند ، صحبت نکنید و در آنها شرکت نکنید. ما از همکاران پروژه های نرم افزار رایگان می خواهیم تا در برابر تعصب و نفرت در پروژه های خود موضع بگیرند. در حالی که این کارها را انجام می دهید ، به این جوامع و FSF بگویید که چرا این کار را انجام می دهید. 
+
+[ما چندین واقعه عمومی راجع به رفتار RMS شرح داده ایم][1]. برخی از ما داستان های خود را در مورد RMS و تعاملات خود با او داریم ، مواردی که در موضوعات ایمیل یا ویدیو ثبت نشده اند. امیدواریم آنچه را که به اشتراک گذاشته شده است بخوانید و آسیبی را که وی به جامعه ما و دیگران وارد کرده است در نظر بگیرید. 
+
+[1]: https://rms-open-letter.github.io/appendix
+
+To sign, please email digitalautonomy at riseup.net or [submit a pull request](https://github.com/rms-open-letter/rms-open-letter.github.io/pulls).
```
Comment (ehashman) [inline index.fa.md:28]: Can you also translate this line?

File: index.fa.md [hunk 5]
```diff
@@ -0,0 +1,2474 @@
...
+
+Signed,
```
Comment (ehashman) [inline index.fa.md:30]: Please drop everything from here down, we aren't keeping copies of the signatures in the translations.

**Review by ehashman — CHANGES_REQUESTED** [review]

File: index.fa.md (hunk 4 above)
Comment (peylight) [reply index.fa.md]: translate added.

File: index.fa.md (hunk 3 above)
Comment (peylight) [reply index.fa.md]: issue has been fixed now.

File: index.fa.md (hunk 2 above)
Comment (peylight) [reply index.fa.md]: Ok, I remove all of these lines.

File: appendix.fa.md (hunk 1 above)
Comment (peylight) [reply appendix.fa.md]: Ok, file has been removed.

File: index.fa.md (hunk 5 above)
Comment (peylight) [reply index.fa.md]: Done.

**Comment by peylight** [comment]
@ehashman
Please check again.

**Comment by ehashman** [comment]
Awesome, thank you :pray:

**Pull request merged**